existing_lints/tokio_time_interval/ui/tokio_time_interval.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2024 Nord Security
+use std::time::Duration;
+use tokio::time::MissedTickBehavior;
+
+const D: Duration = Duration::from_secs(1);
+
+mod interval {
+    use super::*;
+
+    fn incorrect1() {
+        let _interval = tokio::time::interval(D);
+    }
+
+    fn incorrect2() {
+        use tokio::time::interval;
+        let _interval = interval(D);
+    }
+
+    fn different_function_with_same_name() {
+        fn interval(_: Duration) -> u32 {
+            42
+        }
+        let _interval = interval(D);
+    }
+
+    #[allow(tokio_time_interval)]
+    fn correct() {
+        let mut interval = tokio::time::interval(D);
+        interval.set_missed_tick_behavior(MissedTickBehavior::Delay);
```

Comment:
Would be hard to check if next line has missed tick setting?
I would say that resolving complex settup logic some code after settup would not be nice, but
ideomaticaly we should see set_missed_tick_behavour after interval creation, and in such cases it should no longer warn.
For more complex cases, we can leave allow

Reply:
Ok I investigated a bit, and i guess the apis are not really designed for this at lest for an LatePass

Reply:
Yup, I considered a proper lint that would detect that the very next line is setting correct behaviour but I don't see a clear way to implement this without writing lots of code to manage state between different statements being visited.